runelite-api/src/main/java/net/runelite/api/widgets/WidgetInfo.java

```diff
@@ -276,6 +276,7 @@
 	RESIZABLE_VIEWPORT_BOTTOM_LINE_STATS_ICON(WidgetID.RESIZABLE_VIEWPORT_BOTTOM_LINE_GROUP_ID, WidgetID.ResizableViewportBottomLine.SKILLS_ICON),
 	RESIZABLE_VIEWPORT_BOTTOM_LINE_MAGIC_ICON(WidgetID.RESIZABLE_VIEWPORT_BOTTOM_LINE_GROUP_ID, WidgetID.ResizableViewportBottomLine.MAGIC_ICON),
 	RESIZABLE_VIEWPORT_BOTTOM_LINE_FRIEND_ICON(WidgetID.RESIZABLE_VIEWPORT_BOTTOM_LINE_GROUP_ID, WidgetID.ResizableViewportBottomLine.FRIEND_ICON),
+	RESIZABLE_VIEWPORT_BOTTOM_LINE_FRIEND_CHAT_TAB(WidgetID.RESIZABLE_VIEWPORT_BOTTOM_LINE_GROUP_ID, WidgetID.ResizableViewportBottomLine.FC_TAB),
```

Comment:
To match the naming of the other viewports
```suggestion
	RESIZABLE_VIEWPORT_BOTTOM_LINE_FRIENDS_CHAT_TAB(WidgetID.RESIZABLE_VIEWPORT_BOTTOM_LINE_GROUP_ID, WidgetID.ResizableViewportBottomLine.FC_TAB),
```

Reply:
I did it to match the existing `RESIZABLE_VIEWPORT_BOTTOM_LINE_FRIEND_CHAT_ICON`. It's misnamed, but at this point all the `BOTTOM_LINE` viewports are misnamed, given we now know the names are `RESIZABLE_CLASSIC` and `RESIZABLE_MODERN`